src/app/charts/extra-components/index.ts

```diff
@@ -0,0 +1 @@
+export * from './threshold.component';
```

Comment:
It might be nice for this directory to have a more specific name, maybe `extra-params-components` which describes a bit more about the types of components we would expect to see in this directory. `extra-components` is too vague. Can I use this directory for all components that are used within chart component, or is intended just as a folder for the components that provide the "extra indicator params templates"?